src/chrome/content/rules/OASIS_Open.org.xml

```diff
@@ -20,31 +15,21 @@ Fetch error: http://blogs.oasis-open.org/ => https://blogs.oasis-open.org/: (6,
 	Insecure cookies are set for these domains and hosts:
 
 		- .oasis-open.org
-		- .blogs.oasis-open.org
 		- www.oasis-open.org
-
-
-	Mixed content:
-
-		- css on psi from www.oasis-open.org *
-		- Images on psi from www.oasis-open.org *
-
-	* Secured by us
-
 -->
-<ruleset name="OASIS Open.org (partial)" default_off="failed ruleset test">
+<ruleset name="OASIS Open.org (partial)">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="oasis-open.org" />
-	<target host="blogs.oasis-open.org" />
+	<target host="www.oasis-open.org" />
 	<target host="docs.oasis-open.org" />
 	<target host="issues.oasis-open.org" />
 	<target host="lists.oasis-open.org" />
 	<!--target host="psi.oasis-open.org" /-->
+	<target host="support.oasis-open.org" />
 	<target host="tools.oasis-open.org" />
 	<target host="wiki.oasis-open.org" />
-	<target host="www.oasis-open.org" />
 
 
 	<!--	Not secured by server:
```

Comment:
Please remove the `blogs` mention in the securecookie comment since this subdomain is gone from DNS or even better, remove this comment entirely since it's likely outdated.